transport.rb

```diff
@@ -0,0 +1,18 @@
+class Transport
+
+  CONVERT_TO_MINUTES = 60
+
+  attr_accessor :max_weight, :speed, :available
+
+  def initialize(max_weight, speed, available)
```

Comment:
Propose to make the method without defining variables and only `raise NotImplementedError`.
Such trick makes inheritors responsible for defining sensitive data, what could be different. For example max weight for bike or registration_number for car

Reply:
used methods with an exception NotImplementedError